### PR TITLE
refact: update MapsMenu temporal bar

### PR DESCRIPTION
**Pull request by cilasmarques**

### CODE REVIEW

File: src/components/DateInput/DateInput.styles.tsx [hunk 1]
```diff
@@ -17,133 +17,92 @@ export const Wrapper = styled.div<{ disabled: string }>`
...
+export const InputContainer = styled.div`
   width: 100%;
+  height: 2rem;
 `;
 
 export const RangeInput = styled.input`
   width: 100%;
   -webkit-appearance: none;
   background: transparent;
   cursor: pointer;
```
Comment (RodrigoEC) [inline src/components/DateInput/DateInput.styles.tsx:38]:
```suggestion
  cursor: pointer;
  height: 0.25rem;
```

File: src/components/DateInput/DateInput.styles.tsx [hunk 2]
```diff
@@ -17,133 +17,92 @@ export const Wrapper = styled.div<{ disabled: string }>`
...
   &:disabled {
     cursor: not-allowed;
   }
 `;
 
-export const DateList = styled.ul`
+export const DateContainer = styled.div`
   width: 100%;
   display: flex;
   justify-content: space-between;
```
Comment (RodrigoEC) [inline src/components/DateInput/DateInput.styles.tsx:87]:
```suggestion
  align-items: end;
  justify-content: space-between;
  height: 2rem;
```

**Review by RodrigoEC — CHANGES_REQUESTED** [review]
Commented a little fix for chrome and maybe other browsers.

**Review by abraaovalentim — APPROVED** [review]
LGTM

**Review by RodrigoEC — APPROVED** [review]
Very Good!

image

**Pull request merged**